[2022-07-27 19:45:52,914] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v3.check_row_count manual__2022-07-27T19:44:56.114008+00:00 [queued]>
[2022-07-27 19:45:53,002] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v3.check_row_count manual__2022-07-27T19:44:56.114008+00:00 [queued]>
[2022-07-27 19:45:53,002] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 19:45:53,002] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-27 19:45:53,002] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-27 19:45:53,182] {taskinstance.py:1397} INFO - Executing <Task(SnowflakeValueCheckOperator): check_row_count> on 2022-07-27 19:44:56.114008+00:00
[2022-07-27 19:45:53,233] {standard_task_runner.py:52} INFO - Started process 10637 to run task
[2022-07-27 19:45:53,302] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v3', 'check_row_count', 'manual__2022-07-27T19:44:56.114008+00:00', '--job-id', '175', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v3.py', '--cfg-path', '/tmp/tmpvlkkxgns', '--error-file', '/tmp/tmpnqliq_sl']
[2022-07-27 19:45:53,327] {standard_task_runner.py:80} INFO - Job 175: Subtask check_row_count
[2022-07-27 19:45:53,944] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v3.check_row_count manual__2022-07-27T19:44:56.114008+00:00 [running]> on host 2c2de32fb47f
[2022-07-27 19:45:55,198] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v3
AIRFLOW_CTX_TASK_ID=check_row_count
AIRFLOW_CTX_EXECUTION_DATE=2022-07-27T19:44:56.114008+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-27T19:44:56.114008+00:00
[2022-07-27 19:45:55,233] {sql.py:204} INFO - Executing SQL check: SELECT COUNT(*) FROM YELLOW_TRIPDATA;
[2022-07-27 19:45:55,319] {base.py:68} INFO - Using connection ID 'snowflake_default' for task execution.
[2022-07-27 19:45:55,353] {connection.py:262} INFO - Snowflake Connector for Python Version: 2.7.9, Python Version: 3.7.13, Platform: Linux-5.15.0-1014-azure-x86_64-with-debian-11.4
[2022-07-27 19:45:55,363] {connection.py:877} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-07-27 19:45:55,363] {connection.py:894} INFO - Setting use_openssl_only mode to False
[2022-07-27 19:46:00,981] {cursor.py:710} INFO - query: [SELECT COUNT(*) FROM YELLOW_TRIPDATA;]
[2022-07-27 19:46:01,242] {cursor.py:734} INFO - query execution done
[2022-07-27 19:46:01,248] {connection.py:507} INFO - closed
[2022-07-27 19:46:01,481] {connection.py:510} INFO - No async queries seem to be running, deleting session
[2022-07-27 19:46:01,936] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v3, task_id=check_row_count, execution_date=20220727T194456, start_date=20220727T194552, end_date=20220727T194601
[2022-07-27 19:46:02,003] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-27 19:46:02,168] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
